chore(swingset): add tool to measure+display metering usage

warner opened the pull request:
Run `cd packages/SwingSet && node test/measure-metering/measure.js` to get
the report.

This doesn't change the swingset code, it's just a tool that lives in the swingset `test/` directory (but isn't run as part of the tests).

cc @katelynsills

FUDCo reviewed (approved):
Seems fine, but wouldn't this be better placed in the `tools` directory rather than the `test` directory?

warner commented:
Moved to `tools/` and added contract install/instantiate (using the AMM contract). Current report contents:

```
warner@turing:~/stuff/agoric/agoric-sdk/packages/SwingSet$ node tools/measure-metering/measure.js
meterType: xs-meter-10
empty (baseline): 29489
 add async: 98
"let i = 1": 3
"i += 2": 4
"let sum; for (let i=0; i<100; i++) { sum += i; }": 1512
  same but to 1000: 15012
define Far add/read counter object: 39443
  "counter.add(1)": 8

"console.log('')": 1118
makeIssuerKit: 107765
getBrand: 15
getCurrentAmount: 14
deposit: 12383
getUpdateSince: 214
zoe install (treasury): 157677
zoe install (AMM): 168600
zoe instantiate (AMM): 17080161
```
